src/components/admin/HearingToolbar.js

```diff
@@ -38,7 +36,7 @@ class HearingToolbar extends React.Component {
     ];
     const open = isOpen(hearing);
     if (open && hearing.published) {
-      statusLabel = <FormattedMessage id="published"/>;
+      statusLabel = (<Alert bsStyle="success"><Icon name="eye"/> <FormattedMessage id="published"/></Alert>);
```

Comment:
You mean the Alertify popups? I can have a look, I guess they are using the plugin's default style at the moment.